express-post-json/index.js

```diff
@@ -0,0 +1,29 @@
+const express = require('express');
+const app = express();
+
+var nextId = 1;
+var grades = {};
+
+app.listen(3000, () => {
+  // eslint-disable-next-line no-console
+  console.log('Express server listening on port 3000');
+});
+
+app.get('/api/grades', (req, res) => {
+  var gradesArray = [];
+  for (var property in grades) {
+    gradesArray.push(grades[property]);
+  }
+  res.json(gradesArray);
+});
+
+app.use(express.json());
+
+app.post('/api/grades', (req, res) => {
+  res.status(201);
+  var newObj = req.body;
+  newObj.id = nextId;
+  grades[nextId] = newObj;
+  nextId++;
+  res.json(grades);
```

Comment:
Responses to `POST` requests should only contain the newly created grade, not all of them:

---

![image](https://user-images.githubusercontent.com/16143771/142975371-368fb9a8-384d-4d07-9378-05dece0cbcae.png)

Reply:
![image](https://user-images.githubusercontent.com/46618757/143130165-e5256831-49fd-465b-9f8f-c23f339d01fb.png)